src/utils/date.ts

```diff
@@ -126,10 +126,7 @@ export function datePartsFromLocalizedString(
  *
  * @param date
  */
-export function dateToISO(date?: Date | string): string {
-  if (typeof date === "string") {
-    return date;
-  }
+export function dateToISO(date?: Date): string {
```

Comment:
This _shouldn't_ break anything since all usages are sending in `Date` objects or empty strings, and nothing should be relying on the improper value coming back.

Reply:
Yeah typescript would complain if it was recieving a string since you changed the signature. So we should be good